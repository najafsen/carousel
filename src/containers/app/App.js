import React from "react";
import "./App.scss";
import { Main } from "../main/Main";

export const App = () => {
  return (
    <div className="app-component">
      <Main />
    </div>
  );
};
