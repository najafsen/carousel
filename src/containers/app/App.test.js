import React from "react";
import { render } from "@testing-library/react";
import { App } from "./App";

test("matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
