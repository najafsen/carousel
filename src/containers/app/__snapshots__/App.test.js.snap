// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="app-component"
  >
    <div
      class="main-component"
    />
  </div>
</DocumentFragment>
`;
